contrib/Trivy.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+Trivy_container_scanning:
+  stage: test
+  image:
+    name: alpine
```

Comment:
want to make this alpine:3.11 or :latest to be explicit?

Reply:
Let use use `3.11` 👍